### PR TITLE
Bump Restate version to 0.7.2

### PR DESCRIPTION
**Pull request by jackkleeman**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ default-members = [
 resolver = "2"
 
 [workspace.package]
-version = "0.7.1"
+version = "0.7.2"
```
Comment (tillrohrmann) [inline Cargo.toml:20]: I think we should switch back to `0.8.0` because we have breaking changes in `main`.

**Review by tillrohrmann — CHANGES_REQUESTED** [review]
Thanks for creating this PR @jackkleeman. I think we should change to `0.8.0`.

**Comment by tillrohrmann** [comment]
Wait, I actually haven't merged the PR yet where we bump to version `0.8.0`: https://github.com/restatedev/restate/pull/1126. With that PR, we can probably close this PR.

**Comment by jackkleeman** [comment]
OK!

**Pull request closed without merging**